lib/utils/socketChainHelper.js

```diff
@@ -21,18 +20,27 @@ const processServerResponse = toString => function processServerResponseHandler(
 		response: res,
 		toString,
 	});
+	const throwErr = err => {
+		err.stack = data.stack;
+		throw err;
+	};
+	const infoMessage = prefix => getInfoErrorMessage(message, prefix, res, data.stack);
+	const isSuccess = res.result === 'success' || res.result === 'warning';
+	const isEval = res.contentType === 'eval';
+	const isTestLine = res.contentType === 'testLine';
+	const isQuery = res.contentType === 'query';
 
```

Comment:
By these examples you also can create `isWarning` and `isFatal` variables

Reply:
I think these changes come from another PR and are highlighted here just because we are squashing things on merge.
I don't know what the purpose of those is anyway.